.editorconfig

```diff
@@ -1,6 +0,0 @@
-[*]
```

Comment:
Why do you have deleted this file?

Reply:
Yes, please don’t delete it as it’s useful for consistency across different editors